docker-run.sh

```diff
@@ -406,7 +406,7 @@ echo "#"
 echo "# POSTGRES_DIR : $POSTGRES_DIR"
 echo "#---"
 mkdir -p "$POSTGRES_DIR"
-docker run -d --name dev-to-postgres -e POSTGRES_PASSWORD=devto -e POSTGRES_USER=devto -e POSTGRES_DB=PracticalDeveloper_development -v "$POSTGRES_DIR:/var/lib/postgresql/data" postgres:10.7-alpine
+docker run -d --name dev-to-postgres -e POSTGRES_PASSWORD=devto -e POSTGRES_USER=devto -e POSTGRES_DB=PracticalDeveloper_development -v "$POSTGRES_DIR:/var/lib/postgresql/data" postgres:9.6.15-alpine
```

Comment:
maybe we could `postgres:9.6.15-alpine` use the environment variable `POSTGRES_VERSION`.

See https://github.com/thepracticaldev/dev.to/blob/master/docs/installation/docker.md#running-the-docker-app-basic and:

https://github.com/thepracticaldev/dev.to/blob/efbb10806157a220e4d1af1ae21678eb16ec9ed0/docker-compose.yml#L27-L28

Reply:
Yep, makes sense. :+1: